src/pages/Creatures.tsx

```diff
@@ -160,13 +114,13 @@ export default function Creatures() {
                       : FIELDS[field].abbr || FIELDS[field].label
                   }
                   sort={query}
-                  reduceSort={reduceSort}
+                  reduceSort={filterProps.reduceSort}
                 />
               ))}
             </TableRow>
           </TableHead>
           <TableBody>
-            {creatures.map((creature) => (
```

Comment:
Lets still deconstruct results as creatures so this is still page specific. Same for other pages.